### PR TITLE
exim 4.96

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  ==>make
  /bin/sh scripts/source_checks
  
  >>> Creating links to source files...
  ../scripts/Configure-Makefile: 1: ../scripts/Configure-Makefile: pcre2-config: not found
  *** Missing pcre2-config for regular expression support
  Makefile:70: recipe for target 'configure' failed
  make: *** [configure] Error 1
```

File: Formula/exim.rb [hunk 1]
```diff
@@ -40,7 +40,7 @@ class Exim < Formula
 
   depends_on "berkeley-db@4"
   depends_on "openssl@1.1"
-  depends_on "pcre"
+  depends_on "pcre2"
```
Comment (carlocab) [inline Formula/exim.rb:43]:
```suggestion
  depends_on "pcre2"
  uses_from_macos "libxcrypt"
```

**Review by carlocab — COMMENTED** [review]
Website still has 4.95 as the latest version: https://exim.org

**Comment by chenrui333** [comment]
> Website still has 4.95 as the latest version: https://exim.org

that is a good call out, I can reach out to see when that is going to be updated.

**Comment by chenrui333** [comment]
actually, it was announced in the mailing list.
https://lists.exim.org/lurker/message/20220625.141825.d6de6074.en.html

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**